tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/FmiIntegrationTests.cs

```diff
@@ -255,15 +256,16 @@ public async Task Flow5_SubRma_FmiToken_From_FmiCred_For_leafTestAsync()
 
             //Fmi app/scenario parameters
             var clientId = "urn:microsoft:identity:fmi";
-            var scope = "3091264c-7afb-45d4-b527-39737ee86187/.default"; //Guid for api://AzureFMITokenExchange
+            var scope = "3091264c-7afb-45d4-b527-39737ee86187/.default"; 
 
             //Act
             //Create application
             var confidentialApp = ConfidentialClientApplicationBuilder
                         .Create(clientId)
                         .WithAuthority("https://login.microsoftonline.com/f645ad92-e38d-4d1a-b510-d1b09a74a8ca", true)
-                        .WithExtraQueryParameters(_testslice1) //Enables MSAL to target ESTS Test slice
-                        .WithClientAssertion((options) => GetParentCredential(options, _testslice1)) //This api acquires the FMI credential needed to authenticate
+                        .WithExtraQueryParameters(_testslice2) //Enables MSAL to target ESTS Test slice
+                        .WithClientAssertion((options) => GetFmiCredentialFromRma(options, _testslice2)) //This api acquires the FMI credential needed to authenticate
+                        .WithAzureRegion("westus3")
```

Comment:
this can be in a constant to make it easier for future edits. or a method, 
```
private static string GetTestAzureRegion() => "westus3"; // or make it configurable later
```